src/frontend/tsx/Chessboard.tsx

```diff
@@ -42,7 +34,7 @@ declare global {
 // Main
 //
 
-export function Chessboard () {
+export function Chessboard ({pieceTheme, boardTheme}: any) {
```

Comment:
It's considered poor craftsmanship to use the 'any' type in this situation. If we're going to pass the theme info between components, the correct React way to do it would be like this:
```
type ChessboardProps = {
  pieceTheme: string,
  boardTheme: string
}

export function Chessboard ({pieceTheme, boardTheme}: ChessboardProps) {
  ...
```